README.md

```diff
@@ -1,25 +1,198 @@
 
-# generate static project
-$ npm run generate
+A **pre-rendering SPA** is a single page application that generates a static markup (HTML) at build time. The user, when entering the web, receives HTML (as if it were a static web) but in the meantime, JS files belonging to the SPA are loaded “hydrating” the web until it's completely dynamic.
+
+**[Nuxt.js](https://nuxtjs.org/)** is the biggest framework on top of **[Vue.js](https://vuejs.org/)** to generate *universal* SPAs. Universal or "isomorphic" apps can be pre-rendering or SSR. Since so far we don't need server functions, our website it's just pre-rendering.
+
+We create and run unit tests using [Jest](https://jestjs.io/), ensure avoiding syntax errors using [ESLint](https://eslint.org/) and automate all these previous tools and deployment using [Trevis](https://travis-ci.org/).
+
+With this technology we want to **achieve**:
+- Use a component based framework like Vue that allow us to reuse part of the UI code in different parts of the application.
+- Fast initial page load.
+- Index content on Google.
+- Test JS unit functions.
+- Avoid syntax errors.
+- Continuous integration pipeline.
+
+## 🚀 Get Up and Running
+
+
+1. **Download this repository and go to its folder**
+
+    ```shell
+    git clone git@github.com:Qiskit/qiskit.org.git && cd qiskit.org
+    ```
+
+2. **Install dependencies**
+
+    ```shell
+     npm install
+     ```
+
+3. **Run a local server with hot reload at [localhost:3000](localhost:3000)**
+
+    ```shell
+    npm run dev
+    ```
+
+## 🧐 Folder Structure
+
+    .
+    ├── assets
+    ├── components
+    ├── constants
+    ├── content
+    ├── deploy
+    ├── hooks
+    ├── layouts
+    ├── mixins
+    ├── pages
+    ├── plugins
+    ├── static
+    ├── store
+    ├── tests
+    ├── textbook
+    ├── nuxt.config.js
+    ... other third-parties configuration files like ESLint, Jest or Travis
+
+1.  **`/assets`**: You will find the images and assets for the project. You can find more information at [Nuxt's assets directory documentation](https://nuxtjs.org/guide/assets/)
+
+2.  **`/components`**: Vue components for the project. You can find more information at [Nuxt's components directory documentation](https://nuxtjs.org/guide/directory-structure#the-components-directory)
+
+3.  **`/constants`**: Constants shared through the whole project.
+
+4.  **`/content`**: You will save your MD files here. They are divided in folders by sections.
+
+5.  **`/deploy`**: Deploy configuration.
+
+6.  **`/hooks`**: Hook functions shared through the whole project.
+
+7.  **`/layouts`**: You can find information at [Nuxt's layout directory documentation](https://nuxtjs.org/guide/directory-structure#the-layouts-directory)
+
+8.  **`/mixins`**: Mixin functions shared through the whole project.
+
+9.  **`/pages`**: You can find information at [Nuxt's pages directory documentation](https://nuxtjs.org/guide/directory-structure#the-pages-directory)
```

Comment:
Can you mention this is like the point of entry? So if you want to know what is being rendered at any page, you can browse the code inside this folder. It would be nice to mention we use dynamic routes and link https://nuxtjs.org/guide/routing/#dynamic-routes

Reply:
Fixed in the last commit 😊 thanks!